src/components/Pagination/Pagination.tsx

```diff
@@ -20,18 +20,18 @@ export interface PaginationProps extends CorePaginationProps {}
 export const Pagination: React.FunctionComponent<PaginationProps> = props => {
   return (
     <TPAComponentsConsumer>
-      {({ mobile: mobileView, rtl }) => {
+      {({ mobile, rtl }) => {
         return (
           <CorePagination
-            {...styles('root', { mobileView, rtl }, props)}
+            {...styles('root', { mobile, rtl }, props)}
```

Comment:
Theoretically, this is a breaking change (since consumers can manipulate the component's state using stylable).
However, I don't think this will be a problem since I don't think anyone is using this.
I'll check first.

Reply:
Yes, you are right, but I didn't manage to find any other consumers except `Wix Search`

Reply:
@jonathanadler build is green